### PR TITLE
Fix/minor improvements

### PR DESCRIPTION
**Pull request by zkygr**
Hello, 

i updated the example in the readme, because the name of the role was wrong. 
There was an underscore instead of a dash.

I also run in the issue that the mqtt part failed because my `tasmota_mqtt_port` was an integer and not a string and i got no feedback because `no_log` is always `true`. Now the user can switch this to `false`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,15 +137,16 @@ To avoid specifying `tasmota_commands` for each host using host_vars you can use
...
+      hosts: tasmota_devices  
 	  # disable fact gathering since this is currently not possible on tasmota devices  
-      gather_facts: no
+      ansible.builtin.gather_facts: false
       vars:
         tasmota_commands:
             - command: TelePeriod
               value: 10
 	  roles:
-	    - tobias_richter.tasmota
```
Comment (tobias-richter) [inline README.md]: The role name `tobias_richter.tasmota` is absolutely correct, see:  https://galaxy.ansible.com/ui/standalone/roles/tobias_richter/tasmota/

Ansible Galaxy rewrites github namespaces with for example "-" to "-" so `tobias-richter` is transformed to `tobias_richter`. So please revert this change.

**Review by tobias-richter — COMMENTED** [review]
@zkygr thank you for your contribution. I disagree on the role name because it matches the ansible galaxy namespace and the ansible galaxy specs.
But proove me wrong :)

Sp please revert that change and we are good to go.

**Comment by zkygr** [comment]
Hello, and thanks for your fast reply :)

You are absolutly right, i fucked up my import statement in the `requirements.yml`:

```yaml
  - name: tobias-richter.tasmota
    src: https://github.com/tobias-richter/ansible-tasmota
    version: 1.10.0
```
I corrected the readme changes in the squash commit.

**Comment by zkygr** [comment]
Should this role actually be executed on Tasmota devices? Since only the API is being used, it is more likely to be outsourced to localhost, right? That would be the 2nd fixup commit.

**Comment by zkygr** [comment]
If the changes are okay with you, i would squash both commits.

File: README.md [hunk 2]
```diff
@@ -136,15 +137,17 @@ To avoid specifying `tasmota_commands` for each host using host_vars you can use
...
 	  # disable fact gathering since this is currently not possible on tasmota devices  
-      gather_facts: no
+      ansible.builtin.gather_facts: false
       vars:
         tasmota_commands:
             - command: TelePeriod
               value: 10
 	  roles:
 	    - tobias_richter.tasmota
+          delegates_to: localhost
```
Comment (tobias-richter) [inline README.md]: No that is not correct. The action plugin will call the API of the inventory host from localhost. The existing documentation is correct.

**Review by tobias-richter — CHANGES_REQUESTED** [review]
This is my playbook:

```
- hosts: tasmota
  become: no
  gather_facts: no

  roles:
    - tobias_richter.tasmota
```
so no need to delegate to localhost

File: README.md (hunk 2 above)
Comment (tobias-richter) [reply README.md]: Correct would also be `delegate_to`, see also the ansible documentation: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_delegation.html#delegating-tasks

Comment (tobias-richter) [reply README.md]: If you would have tested your documentation changes you would have seen that it is not working with delegation to localhost:
```
PLAY [tasmota] ******************************************************************************************************

TASK [tobias_richter.tasmota : Configure tasmota.] ******************************************************************
Thursday 07 December 2023  22:00:53 +0100 (0:00:00.158)       0:00:00.158 *****
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /cm?user=&password=&cmnd=NtpServer1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3203cb8f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
fatal: [socket055]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}

PLAY RECAP **********************************************************************************************************
socket055     : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

Thursday 07 December 2023  22:00:53 +0100 (0:00:00.041)       0:00:00.200 *****
===============================================================================
tobias_richter.tasmota : Configure tasmota. ------------------------------------------------------------------ 0.04s
```

Comment (zkygr) [reply README.md]: Happy new year :D

I tested my configuration changes, but i set `ansible_host` in the inventory and it worked.
Nevermind, you are  correct, `delegate_to` is not needed.
I deleted the documentation changes.

**Review by tobias-richter — APPROVED** [review]
